pkgs/by-name/tc/tcptrace/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  stdenv,
+  fetchurl,
+  lib,
+  libpcap,
+}:
+stdenv.mkDerivation (final: {
+  name = "tcptrace";
+  version = "6.6.7";
+
+  srcs = [
+    (fetchurl {
+      # Home page was down on 2024-02-01, so use Debian mirror as fallback.
+      urls = [
+        "http://www.tcptrace.org/download/tcptrace-${final.version}.tar.gz"
+        "mirror://debian/pool/main/t/tcptrace/tcptrace_${final.version}.orig.tar.gz"
+      ];
+      hash = "sha256-YzgKQFGTPKCJeUdqnfxvlZMIvJ9g1FJVIC44jrVpEL0=";
+    })
+    (fetchurl {
+      url = "mirror://debian/pool/main/t/tcptrace/tcptrace_6.6.7-6.debian.tar.xz";
+      hash = "sha256-8rkwzdXcDOc3kACvrlyTQsnSw6AJgy6xA0YrECu63gY=";
+    })
```

Comment:
Build does not work with this source archive.

Reply:
Can you tell me more about what you're seeing? The build is working fine for me...

Reply:
If I remove the first entry in src, the build fails. I will post the full log tomorrow

Reply:
Because the first entry is the source code, and this second entry is patches from Debian, not the source code, right?

Reply:
Oh, I must have missed that. Nevermind